contracts/TestDKVDaggerHashimoto.sol

```diff
@@ -89,4 +89,9 @@ contract TestDecentralizedKVDaggerHashimoto is DecentralizedKVDaggerHashimoto {
     ) public {
         _rewardMiner(startShardId, shardLen, miner, minedTs, diffs, hash0);
     }
+
+    function unmaskWithEthash(
+        uint256 kvIdx,
+        bytes memory maskedData
+    ) external view returns (bytes memory){}
```

Comment:
Ditto. Really qustioning the necessity of existence of this function.......

Reply:
Yeap, but some tests and code still dependency this function.